store/mockstore/unistore/cophandler/closure_exec.go

```diff
@@ -510,11 +572,18 @@ func (e *tableScanProcessor) Process(key, value []byte) error {
 	if e.rowCount == e.limit {
 		return dbreader.ScanBreak
 	}
+	gotRow := false
+	defer func(begin time.Time) {
+		e.scanCtx.execDetail.update(begin, gotRow)
+	}(time.Now())
 	e.rowCount++
-	err := e.tableScanProcessCore(key, value)
+	err := e.tableScanProcessCore(key, value, false)
```

Comment:
How about setting it to `true`?
Since if an error occurs here, it will be returned to `TiDB` and the user will get this error, so execution information makes no then.

Reply:
I don't quite understand what you mean.
This boolean variable is only used to determine whether to collect `execDetail`, this is to avoid repeated collection by `tableScan`